frontend/views/containers/chatroom/MessageActions.vue

```diff
@@ -164,10 +164,11 @@ export default ({
       return this.isText || this.isPoll
     },
     isEditable () {
-      return this.isMsgSender && (this.isText || this.isPoll)
+      return this.isMsgSender && this.isText
     },
     isDeletable () {
-      return this.isEditable || this.isGroupCreator
+      return this.isGroupCreator ||
+        (this.isMsgSender && (this.isText || this.isPoll))
```

Comment:
Is there a situation where we wouldn't want a message to be deletable when `this.isMsgSender` is true?

Reply:
@taoeffect Yes there is. This kind of notification message is one of that example.

![image](https://github.com/okTurtles/group-income/assets/17641213/5f39c6d7-626f-4a84-8f00-1637d73d5afb)